reminders/apps.py

```diff
@@ -1,6 +1,19 @@
+from threading import Thread
 from django.apps import AppConfig
 
 
 class RemindersConfig(AppConfig):
     default_auto_field = 'django.db.models.BigAutoField'
     name = 'reminders'
+
+    scheduler = None
+
+    def ready(self) -> None:
+        if not self.scheduler:
+            Thread(target=RemindersConfig.create_scheduler, daemon=True).start()
+
+    @staticmethod
+    def create_scheduler():
+        from reminders.scheduler import UserAlertScheduler
+
+        RemindersConfig.scheduler = UserAlertScheduler()
```

Comment:
if you want to update the scheduler maybe use the self like:
```suggestion
    def create_scheduler(self):
        from reminders.scheduler import UserAlertScheduler

        self.scheduler = UserAlertScheduler()
```

Reply:
if I would do that I would need to send the self variable to the create scheduler function,
In my opinion, this is unnecessary because this is a variable of the class.